src/main/webapp/chart.html

```diff
@@ -1,48 +1,16 @@
 <!DOCTYPE html>
 
 <html>
-  <head>
-    <script
-      type="text/javascript"
-      src="https://www.gstatic.com/charts/loader.js"
-    ></script>
-    //TODO move all of this to a separate JavaScript file
-    //figure out a way to get user input/data for the charts
-    //make sure to implement same idea as in the stats page
-    <script type="text/javascript">
-      google.charts.load("current", { packages: ["corechart"] });
-      google.charts.setOnLoadCallback(drawChart);
-      function drawChart() {
-        var book_data = new google.visualization.DataTable();
-        //define columns for the DataTable instance
-        book_data.addColumn("string", "Book Title");
-        book_data.addColumn("number", "Votes");
 
-        //add data to book_data
-        book_data.addRows([
-          ["The Best We Could Do", 6],
-          ["Sing, Unburied, Sing", 10],
-          ["The Book of Unknown Americans", 7],
-          ["The 57 Bus", 4],
-          ["The Handmaid's Tale", 8]
-        ]);
-        var chart = new google.visualization.BarChart(
-          document.getElementById("book_chart")
-        );
+<head>
+  <script src="https://www.gstatic.com/charts/loader.js"></script>
+  <script src="/js/chart-loader.js"></script>
+  <title>A New Chart</title>
+</head>
 
-        //optional customization as dictionary
-        var chart_options = {
-          width: 800,
-          height: 400
-        };
-        chart.draw(book_data, chart_options);
-      }
-    </script>
-    <title>A New Chart</title>
-  </head>
+<body>
+  <h1>Chart Project</h1>
+  <div id="bookChart"></div>
```

Comment:
Minor nit: although it is standard in JavaScript to use `camelCase`, it is more common in CSS identifiers to use `lower-case-with-hyphens` for `id` and `class` attributes.

Reply:
Please address this one last comment before submitting.

Reply:
updated CSS identifiers to use lower-case-with-hyphens